src/test/java/seedu/address/logic/commands/EditStudentDescriptorTest.java

```diff
@@ -12,15 +12,17 @@
 
 import org.junit.jupiter.api.Test;
 
-import seedu.address.logic.commands.EditCommand.EditPersonDescriptor;
-import seedu.address.testutil.EditPersonDescriptorBuilder;
+import seedu.address.logic.commands.student.EditStudentCommand;
+import seedu.address.logic.commands.student.EditStudentCommand.EditStudentDescriptor;
+import seedu.address.testutil.EditStudentDescriptorBuilder;
 
 public class EditStudentDescriptorTest {
 
     @Test
     public void equals() {
         // same values -> returns true
-        EditPersonDescriptor descriptorWithSameValues = new EditPersonDescriptor(DESC_AMY);
+        EditStudentCommand.EditStudentDescriptor descriptorWithSameValues =
+                new EditStudentCommand.EditStudentDescriptor(DESC_AMY);
```

Comment:
should just use `EditStudentDescriptor `directly instead of referring to `EditStudentCommand`